templates/Simple/style.css

```diff
@@ -100,7 +100,7 @@ table.head {
 }
 
 td.header img {
-	max-height: 3cm;
+	/* max-height: 3cm; */
```

Comment:
I would leave this in as default, maybe with an annotation that it is overriden by the settings by default